pom.xml

```diff
@@ -1165,7 +1165,6 @@
       <groupId>org.flywaydb</groupId>
       <artifactId>flyway-play_${scala.package.version}</artifactId>
       <version>${flyway.play.version}</version>
-      <scope>runtime</scope>
```

Comment:
Scope changed because I need to block on the flyway-play `PlayInitializer` from within `MainModule` to guarantee that migrations have run.